integration/data/common.py

```diff
@@ -31,9 +31,15 @@
 UPGRADE_REPLICA2 = 'tcp://localhost:9515'
 UPGRADE_REPLICA2_SCHEMA = 'http://localhost:9515/v1/schemas'
 
-CONTROLLER = "http://localhost:9501"
+STANDBY_REPLICA1 = 'tcp://localhost:9522'
+STANDBY_REPLICA1_SCHEMA = 'http://localhost:9522/v1/schemas'
+STANDBY_REPLICA2 = 'tcp://localhost:9525'
+STANDBY_REPLICA2_SCHEMA = 'http://localhost:9525/v1/schemas'
+
+STANDBY_REPLICA1_PATH = '/tmp/fix1/'
```

Comment:
`fix1` is weird. How about `/tmp/standby_vol1`?

And this should be in the next commit?

Reply:
vendor file commit is the 1st first. met github bug again...
In engine command, the short name of flag `--incrementally` is `-i`. Is that OK?

Reply:
I think you made it `-I`, but either is OK.

Reply:
Double check.